erpnext/accounts/utils.py

```
@@ -1544,7 +1553,7 @@ def parse_naming_series_variable(doc, variable):
 
 
 @frappe.whitelist()
-def get_coa(doctype, parent, is_root=None, chart=None):
+def get_coa(doctype: str, parent: str, is_root: bool | None = None, chart: str | None = None):
```

Ruff (0.15.2)

[warning] Unused function argument: `doctype`

(ARG001)